dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/TldParser.java

```diff
@@ -82,6 +83,17 @@
     
     public static final String WEB_SERVICE_CLIENT_XSD_SYSTEM_ID = "http://www.ibm.com/webservices/xsd/j2ee_web_services_client_1_1.xsd";
     public static final String WEB_SERVICE_CLIENT_XSD_RESOURCE_PATH = "/javax/servlet/resources/j2ee_web_services_client_1_1.xsd";
+
+    /* Servlet 5.0 API contains these JSP files */
+
+    public static final String EE9_TAGLIB_DTD_PUBLIC_ID_11 = "-//Sun Microsystems, Inc.//DTD JSP Tag Library 1.1//EN";
+    public static final String EE9_TAGLIB_DTD_RESOURCE_PATH_11 = "/jakarta/servlet/resources/web-jsptaglibrary_1_1.dtd";
+    
+    public static final String EE9_TAGLIB_DTD_PUBLIC_ID_12 = "-//Sun Microsystems, Inc.//DTD JSP Tag Library 1.2//EN";
+    public static final String EE9_TAGLIB_DTD_RESOURCE_PATH_12 = "/jakarta/servlet/resources/web-jsptaglibrary_1_2.dtd";
+
+    public static final String EE9_TAGLIB_XSD_SYSTEM_ID_20 = "web-jsptaglibrary_2_0.xsd";
+    public static final String EE9_TAGLIB_XSD_RESOURCE_PATH_20 = "/jakarta/servlet/resources/web-jsptaglibrary_2_0.xsd";
```

Comment:
The following don't change, so many we shouldn't duplicate these values. Only make add variables for the new paths? 

- EE9_TAGLIB_DTD_PUBLIC_ID_11
- EE9_TAGLIB_DTD_PUBLIC_ID_12
- EE9_TAGLIB_XSD_SYSTEM_ID_20